npu.user.js

```diff
@@ -985,9 +989,43 @@ var npu = {
 		
 	/* == EXAM LIST == */
 	
+		/* Decide whether the given grade string constitutes passing the exam */
+		isPassingGrade: function(gradeStr) {
+			if (npu.getLanguage() == "hu") {

```

Comment:
Why fetch the language in the first place? I would simply do it like this:
`["Elégtelen", "Nem felelt meg", "Nem jelent meg", "Fail", "Did not attend"].indexOf(gradeStr) == -1`